### ISSUE TITLE
Ruby Gem install Json fails on Mavericks and Xcode 5.1 - unknown argument: '-multiply_definedsuppress'

### ISSUE THREAD

**Issue by henry74** [boxen/our-boxen#528]
Getting this error when trying to run boxen on Maverick and latest developer tools - same as this stackoverflow question (http://stackoverflow.com/questions/22352838/ruby-gem-install-json-fails-on-mavericks-and-xcode-5-1-unknown-argument-mul) which at this time is 3 hours old:
```
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby extconf.rb 
creating Makefile

make "DESTDIR=" clean

make "DESTDIR="
compiling generator.c
linking shared-object json/ext/generator.bundle
clang: error: unknown argument: '-multiply_definedsuppress' [-Wunused-command-line-argument-hard-error-in-future]
clang: note: this will be a hard error (cannot be downgraded to a warning) in the future
make: *** [generator.bundle] Error 1

make failed, exit code 2

Gem files will remain installed in /opt/boxen/repo/.bundle/ruby/2.0.0/gems/json-1.8.0 for inspection.
Results logged to /opt/boxen/repo/.bundle/ruby/2.0.0/extensions/universal-darwin-13/2.0.0/json-1.8.0/gem_make.out
```

**Comment by petems** [comment]
This issue still exists (only on older machines, on my OSX VM it happens)

PR to fix it: https://github.com/boxen/our-boxen/pull/712

**Comment by seanknox** [comment]
This was fixed in be298839fa82dcbbf94ebd0032de8667d1694009 and later improved in 056ca871.
